ManPhamQuang/COSC2769-rmit
# Feat/user edit

### Changelog
- Added Edit User form
- Added new Auth Reducer Actions (GETME_START, GETMESUCCESS, GETME_FAILURE)
- Refactored Layout Implementation (fixes #86)

### How to test
- Checkout PR
- Login with either an expert account (madmax@example.com/12345678) or user account (sadmax@gmail.com/12345678)
- Click on the edit profile button located in the user dropdown menu.
- Verify that the app navigate to different screen for different user role (eg. the expert account should take you to the profile screen in dashboard)
- Fill out the edit form and press the Save button
- Verify that the User/Expert information is updated after the process is finished
